javascript/javascript1/week4/creditCardApp.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+function formatCreditCardNumber () {
+  const digits= "(975318642359)";
+  const newDigits = "(" + digits.match(/(\d{4})/g).join(" ") + ")"; //(9753 1864 2359)
+  console.log(newDigits);
+};
```

Comment:
Based on how you're using this function below, you should be passing digits in as a parameter to your function so you can use it anywhere! Function parameters are what make our functions versatile and reusable. 
```suggestion
function formatCreditCardNumber (digits) {
  const newDigits = "(" + digits.match(/(\d{4})/g).join(" ") + ")"; //(9753 1864 2359)
return newDigits
};
```